src/Input/Input.js

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from '../utils/customPropTypes';
 
-function isDirty(obj) {
+export function isDirty(obj) {
```

Comment:
Sorry, missed this question. The name is fine, I just can't decide if function warrants sharing given it's so small, but rather just duplicating it across the two components.

If it was going to be used more broadly then maybe. As it stands, I'm not so sure.